src/ralph/networks/admin.py

```diff
@@ -208,6 +224,12 @@ def show_addresses(self, network):
     show_addresses.allow_tags = True
     show_addresses.short_description = _('Addresses')
 
+    def get_queryset(self, request):
+        qs = super().get_queryset(request)
+        qs = qs.annotate(ipaddress_count=Count('ips'))
+        qs = qs.extra(select={'subnetwork_count': 'rght - lft'})
```

Comment:
add a comment that this is not exact number of subnetworks but it's fine estimate for ordering